sdk/core/Azure.Core.TestFramework/src/TestLogger.cs

```diff
@@ -90,7 +90,7 @@ public void LogEvent(EventWrittenEventArgs args)
             // Add a line to separate requests/responses within a single test
             message.AppendLine();
             // Output the request/response
-            TestContext.Out.WriteLine(message.ToString());
+            TestContext.Error.WriteLine(message.ToString());
```

Comment:
Is this correct? Should we condition log on the test failing?

Reply:
I was hoping it would just work by not including std error in the console.
I think things worked due to a bug before, so now we do need to fix it to condition on the test failing.